Native build leads to HibernateException: Unable to parse order-by fragment

**Describe the bug**
Adding a `@OrderBy` annotation to a `@OneToMany` mapping leads to a exception in Hibernate with native compilation.

**Expected behavior**
Working native compilation. 

**Actual behavior**
See 'To Reproduce'


**To Reproduce**
Steps to reproduce the behavior:
1. run 'mvn io.quarkus:quarkus-maven-plugin:0.22.0:create'
1. run 'cd my-quarkus-project'
2. run './mvnw quarkus:add-extension -Dextensions="jpa"' 'io.quarkus:quarkus-hibernate-orm' is added
3. run '/mvnw quarkus:add-extension -Dextensions="io.quarkus:quarkus-jdbc-postgresql"'
3. add 
```java
package org.acme.quarkus.sample;

import java.util.List;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.OrderBy;

@Entity
public class Parent {

	@Id
	private Long id;

	@OneToMany(mappedBy = "parent")
	@OrderBy("forename")
	private List<Child> children;

	public List<Child> getChildren() {
		return children;
	}

	public void setChildren(List<Child> children) {
		this.children = children;
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

}
```
3. Add 
```java
package org.acme.quarkus.sample;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.ManyToOne;

@Entity
public class Child {

	@Id
	private Long id;

	@ManyToOne
	private Parent parent;

	private String forename;

	private String favoritePet;

	public Parent getParent() {
		return parent;
	}

	public void setParent(Parent parent) {
		this.parent = parent;
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getForename() {
		return forename;
	}

	public void setForename(String forename) {
		this.forename = forename;
	}

	public String getFavoritePet() {
		return favoritePet;
	}

	public void setFavoritePet(String favoritePet) {
		this.favoritePet = favoritePet;
	}

}
```
3. run 'mvn clean package -DskipTests -Pnative -Dnative-image.docker-build=true && docker build -f src/main/docker/Dockerfile.native -t quarkus/honey . && docker run -i --rm -p 8080:8080 quarkus/honey'
3. Console says: 
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
[INFO] 
[INFO] -------------< org.acme.quarkus.sample:my-quarkus-project >-------------
[INFO] Building my-quarkus-project 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ my-quarkus-project ---
[INFO] Deleting /home/jonas/Documents/my-quarkus-project/target
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ my-quarkus-project ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ my-quarkus-project ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 3 source files to /home/jonas/Documents/my-quarkus-project/target/classes
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ my-quarkus-project ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/jonas/Documents/my-quarkus-project/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ my-quarkus-project ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 2 source files to /home/jonas/Documents/my-quarkus-project/target/test-classes
[INFO] 
[INFO] --- maven-surefire-plugin:2.22.0:test (default-test) @ my-quarkus-project ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ my-quarkus-project ---
[INFO] Building jar: /home/jonas/Documents/my-quarkus-project/target/my-quarkus-project-1.0-SNAPSHOT.jar
[INFO] 
[INFO] --- quarkus-maven-plugin:0.22.0:build (default) @ my-quarkus-project ---
[INFO] [io.quarkus.deployment.QuarkusAugmentor] Beginning quarkus augmentation
[INFO] [org.jboss.threads] JBoss Threads version 3.0.0.Beta5
[INFO] [org.hibernate.jpa.boot.internal.PersistenceXmlParser] HHH000318: Could not find any META-INF/persistence.xml file in the classpath
[INFO] [org.hibernate.Version] HHH000412: Hibernate Core {5.4.4.Final}
[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 804ms
[INFO] [io.quarkus.creator.phase.runnerjar.RunnerJarPhase] Building jar: /home/jonas/Documents/my-quarkus-project/target/my-quarkus-project-1.0-SNAPSHOT-runner.jar
[INFO] 
[INFO] --- quarkus-maven-plugin:0.22.0:native-image (default) @ my-quarkus-project ---
[INFO] [io.quarkus.creator.phase.nativeimage.NativeImagePhase] Running Quarkus native-image plugin on OpenJDK 64-Bit Server VM
[INFO] [io.quarkus.creator.phase.nativeimage.NativeImagePhase] docker run -v /home/jonas/Documents/my-quarkus-project/target:/project:z --rm --user 1000:1000 quay.io/quarkus/ubi-quarkus-native-image:19.1.1 -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime -jar my-quarkus-project-1.0-SNAPSHOT-runner.jar -J-Djava.util.concurrent.ForkJoinPool.common.parallelism=1 -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:+PrintAnalysisCallTree -H:-AddAllCharsets -H:EnableURLProtocols=http,https --enable-all-security-services -H:-SpawnIsolates -H:+JNI --no-server -H:-UseServiceLoaderFeature -H:+StackTrace
[my-quarkus-project-1.0-SNAPSHOT-runner:27]    classlist:   4,762.01 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]        (cap):     756.84 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]        setup:   1,687.24 ms
18:39:24,457 INFO  [org.hib.Version] HHH000412: Hibernate Core {5.4.4.Final}
18:39:24,479 INFO  [org.hib.ann.com.Version] HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
18:39:24,504 INFO  [org.hib.dia.Dialect] HHH000400: Using dialect: io.quarkus.hibernate.orm.runtime.dialect.QuarkusPostgreSQL95Dialect
18:39:25,694 INFO  [org.jbo.threads] JBoss Threads version 3.0.0.Beta5
18:39:26,122 INFO  [org.xnio] XNIO version 3.7.2.Final
18:39:26,160 INFO  [org.xni.nio] XNIO NIO Implementation Version 3.7.2.Final
18:39:26,161 INFO  [com.arj.ats.arjuna] ARJUNA012170: TransactionStatusManager started on port 41275 and host 127.0.0.1 with service com.arjuna.ats.arjuna.recovery.ActionStatusService
[my-quarkus-project-1.0-SNAPSHOT-runner:27]   (typeflow):  13,220.70 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]    (objects):   9,395.24 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]   (features):     858.73 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]     analysis:  24,626.50 ms
Printing call tree to /project/reports/call_tree_my-quarkus-project-1.0-SNAPSHOT-runner_20190929_183954.txt
Printing list of used classes to /project/reports/used_classes_my-quarkus-project-1.0-SNAPSHOT-runner_20190929_183956.txt
Printing list of used packages to /project/reports/used_packages_my-quarkus-project-1.0-SNAPSHOT-runner_20190929_183956.txt
[my-quarkus-project-1.0-SNAPSHOT-runner:27]     (clinit):     460.03 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]     universe:   1,270.78 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]      (parse):   1,943.67 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]     (inline):   3,340.55 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]    (compile):  18,995.41 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]      compile:  25,704.00 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]        image:   2,663.15 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]        write:     480.47 ms
[my-quarkus-project-1.0-SNAPSHOT-runner:27]      [total]:  69,728.49 ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:14 min
[INFO] Finished at: 2019-09-29T20:40:27+02:00
[INFO] ------------------------------------------------------------------------
Sending build context to Docker daemon  65.22MB
Step 1/6 : FROM registry.access.redhat.com/ubi8/ubi-minimal
 ---> 9c9faa7498e5
Step 2/6 : WORKDIR /work/
 ---> Using cache
 ---> 5227041ae6e3
Step 3/6 : COPY target/*-runner /work/application
 ---> 8d03b10c3d85
Step 4/6 : RUN chmod 775 /work
 ---> Running in f02b27f48439
Removing intermediate container f02b27f48439
 ---> de91a439e294
Step 5/6 : EXPOSE 8080
 ---> Running in bfc514f266f9
Removing intermediate container bfc514f266f9
 ---> 8f33902d2c1e
Step 6/6 : CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
 ---> Running in 75f7b81c5f2a
Removing intermediate container 75f7b81c5f2a
 ---> 19109c9fc52a
Successfully built 19109c9fc52a
Successfully tagged quarkus/honey:latest
Can't create AST Node org.hibernate.sql.ordering.antlr.NodeSupport
javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
	at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
	at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits18.deploy_0(HibernateOrmProcessor$startPersistenceUnits18.zig:51)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits18.deploy(HibernateOrmProcessor$startPersistenceUnits18.zig:70)
	at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:90)
	at io.quarkus.runtime.Application.start(Application.java:91)
	at io.quarkus.runtime.Application.run(Application.java:204)
	at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:34)
Caused by: org.hibernate.MappingException: Could not get constructor for org.hibernate.persister.collection.OneToManyPersister
	at org.hibernate.persister.internal.PersisterFactoryImpl.createCollectionPersister(PersisterFactoryImpl.java:178)
	at org.hibernate.persister.internal.PersisterFactoryImpl.createCollectionPersister(PersisterFactoryImpl.java:140)
	at org.hibernate.metamodel.internal.MetamodelImpl.initialize(MetamodelImpl.java:225)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:300)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
	... 12 more
Caused by: org.hibernate.HibernateException: Unable to parse order-by fragment
	at org.hibernate.sql.ordering.antlr.OrderByFragmentTranslator.translate(OrderByFragmentTranslator.java:55)
	at org.hibernate.sql.Template.translateOrderBy(Template.java:724)
	at org.hibernate.persister.collection.AbstractCollectionPersister.<init>(AbstractCollectionPersister.java:571)
	at org.hibernate.persister.collection.OneToManyPersister.<init>(OneToManyPersister.java:69)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.hibernate.persister.internal.PersisterFactoryImpl.createCollectionPersister(PersisterFactoryImpl.java:152)
	... 17 more
Caused by: java.lang.NullPointerException
	at org.hibernate.sql.ordering.antlr.OrderByFragmentParser.isFunctionName(OrderByFragmentParser.java:74)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.expression(GeneratedOrderByFragmentParser.java:653)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.sortKey(GeneratedOrderByFragmentParser.java:347)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.sortSpecification(GeneratedOrderByFragmentParser.java:242)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.orderByFragment(GeneratedOrderByFragmentParser.java:191)
	at org.hibernate.sql.ordering.antlr.OrderByFragmentTranslator.translate(OrderByFragmentTranslator.java:49)
	... 22 more
Exception in thread "main" java.lang.RuntimeException: Failed to start quarkus
	at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:126)
	at io.quarkus.runtime.Application.start(Application.java:91)
	at io.quarkus.runtime.Application.run(Application.java:204)
	at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:34)
Caused by: javax.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:113)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:67)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:54)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:79)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:54)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:109)
	at io.quarkus.hibernate.orm.runtime.JPAConfig.startAll(JPAConfig.java:57)
	at io.quarkus.hibernate.orm.runtime.HibernateOrmRecorder.startAllPersistenceUnits(HibernateOrmRecorder.java:77)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits18.deploy_0(HibernateOrmProcessor$startPersistenceUnits18.zig:51)
	at io.quarkus.deployment.steps.HibernateOrmProcessor$startPersistenceUnits18.deploy(HibernateOrmProcessor$startPersistenceUnits18.zig:70)
	at io.quarkus.runner.ApplicationImpl1.doStart(ApplicationImpl1.zig:90)
	... 3 more
Caused by: org.hibernate.MappingException: Could not get constructor for org.hibernate.persister.collection.OneToManyPersister
	at org.hibernate.persister.internal.PersisterFactoryImpl.createCollectionPersister(PersisterFactoryImpl.java:178)
	at org.hibernate.persister.internal.PersisterFactoryImpl.createCollectionPersister(PersisterFactoryImpl.java:140)
	at org.hibernate.metamodel.internal.MetamodelImpl.initialize(MetamodelImpl.java:225)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:300)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:462)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:65)
	... 12 more
Caused by: org.hibernate.HibernateException: Unable to parse order-by fragment
	at org.hibernate.sql.ordering.antlr.OrderByFragmentTranslator.translate(OrderByFragmentTranslator.java:55)
	at org.hibernate.sql.Template.translateOrderBy(Template.java:724)
	at org.hibernate.persister.collection.AbstractCollectionPersister.<init>(AbstractCollectionPersister.java:571)
	at org.hibernate.persister.collection.OneToManyPersister.<init>(OneToManyPersister.java:69)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.hibernate.persister.internal.PersisterFactoryImpl.createCollectionPersister(PersisterFactoryImpl.java:152)
	... 17 more
Caused by: java.lang.NullPointerException
	at org.hibernate.sql.ordering.antlr.OrderByFragmentParser.isFunctionName(OrderByFragmentParser.java:74)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.expression(GeneratedOrderByFragmentParser.java:653)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.sortKey(GeneratedOrderByFragmentParser.java:347)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.sortSpecification(GeneratedOrderByFragmentParser.java:242)
	at org.hibernate.sql.ordering.antlr.GeneratedOrderByFragmentParser.orderByFragment(GeneratedOrderByFragmentParser.java:191)
	at org.hibernate.sql.ordering.antlr.OrderByFragmentTranslator.translate(OrderByFragmentTranslator.java:49)
	... 22 more
```

**Configuration**
```properties
quarkus.datasource.url=jdbc:postgresql://192.168.2.109:32768/postgres
quarkus.datasource.driver=org.postgresql.Driver
quarkus.datasource.username=postgres
quarkus.datasource.password=mysecretpassword
quarkus.datasource.min-size=3
quarkus.datasource.max-size=13

quarkus.hibernate-orm.database.generation=update
#quarkus.hibernate-orm.database.generation=drop-and-create
quarkus.hibernate-orm.log.sql=true
```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - `uname -a`: `Linux jonas-Z170-HD3P 5.0.0-29-generic #31-Ubuntu SMP Thu Sep 12 13:05:32 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`
 - Output of `java -version`: 
```
openjdk version "11.0.4" 2019-07-16
OpenJDK Runtime Environment (build 11.0.4+11-post-Ubuntu-1ubuntu219.04)
OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Ubuntu-1ubuntu219.04, mixed mode, sharing)
``` 
 - GraalVM version (if different from Java): GraalVM version from docker image?
 - Quarkus version: `0.22.0`

**Additional context**
As this is my first bug ticket in github I am keen on advice how to right a good one.

--- a/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmReflections.java
+++ b/extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmReflections.java
@@ -127,6 +127,14 @@ public final class HibernateOrmReflections {
         simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.BinaryArithmeticOperatorNode.class);
         simpleConstructor(reflectiveClass, CommonToken.class);
 
+        // Support for @OrderBy
+        simpleConstructor(reflectiveClass, org.hibernate.sql.ordering.antlr.NodeSupport.class);
+        simpleConstructor(reflectiveClass, org.hibernate.sql.ordering.antlr.OrderByFragment.class);
+        simpleConstructor(reflectiveClass, org.hibernate.sql.ordering.antlr.SortSpecification.class);
+        simpleConstructor(reflectiveClass, org.hibernate.sql.ordering.antlr.OrderingSpecification.class);
+        simpleConstructor(reflectiveClass, org.hibernate.sql.ordering.antlr.CollationSpecification.class);
+        simpleConstructor(reflectiveClass, org.hibernate.sql.ordering.antlr.SortKey.class);
+
         //The CoreMessageLogger is sometimes looked up without it necessarily being a field, so we're
         //not processing it the same way as other Logger lookups.
         simpleConstructor(reflectiveClass, "org.hibernate.internal.CoreMessageLogger_$logger");

--- a/integration-tests/jpa/src/main/java/io/quarkus/it/jpa/orderbyfragment/Child.java
+++ b/integration-tests/jpa/src/main/java/io/quarkus/it/jpa/orderbyfragment/Child.java
@@ -1,0 +1,52 @@
+package io.quarkus.it.jpa.orderbyfragment;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Child {
+
+    @Id
+    private Long id;
+
+    @ManyToOne
+    private Parent parent;
+
+    private String forename;
+
+    private String favoritePet;
+
+    public Parent getParent() {
+        return parent;
+    }
+
+    public void setParent(Parent parent) {
+        this.parent = parent;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getForename() {
+        return forename;
+    }
+
+    public void setForename(String forename) {
+        this.forename = forename;
+    }
+
+    public String getFavoritePet() {
+        return favoritePet;
+    }
+
+    public void setFavoritePet(String favoritePet) {
+        this.favoritePet = favoritePet;
+    }
+
+}

--- a/integration-tests/jpa/src/main/java/io/quarkus/it/jpa/orderbyfragment/Parent.java
+++ b/integration-tests/jpa/src/main/java/io/quarkus/it/jpa/orderbyfragment/Parent.java
@@ -1,0 +1,36 @@
+package io.quarkus.it.jpa.orderbyfragment;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+
+@Entity
+public class Parent {
+
+    @Id
+    private Long id;
+
+    @OneToMany(mappedBy = "parent")
+    @OrderBy("forename")
+    private List<Child> children;
+
+    public List<Child> getChildren() {
+        return children;
+    }
+
+    public void setChildren(List<Child> children) {
+        this.children = children;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+}

